docker-compose.yml

```diff
@@ -0,0 +1,52 @@
+peer:
+        image: 'desdemona:0.1.0'
+        links:
+          - zookeeper:zk
+          - kafka:kafka
+          - db:db
+          - syslog-ng:syslog-ng
+        environment:
+          ONYX_ID: 1
+          NPEERS: 6
+        privileged: true
+        expose:
+          - "40200"
+          - "40200/udp"
+zookeeper:
+        image: 'wurstmeister/zookeeper'
+        ports:
+          - '2181:2181'
+kafka:
+        image: 'wurstmeister/kafka'
+        environment:
+          KAFKA_BROKER_ID: 1
+          HOSTNAME_COMMAND: "route -n | awk '/UG[ \t]/{print $$2}'"
+        links:
+          - zookeeper:zk
+        ports:
+          - "9092:9092"
+        volumes:
+          - /var/run/docker.sock:/var/run/docker.sock
+db:
```

Comment:
This should also go away of course, but I guess that's what #8 is about, and it's better to have this for now?